submissions/Olumide Ologbon/html/signup.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            signup
+        </title>  
+        <link href="style.css" rel="stylesheet">
+    </head>
+    <header class="head">
+        <div>
+            <div class="dropdown">
+                <button class="dropbtn">More</button>
+                <div class="dropdown-ctn">
+                    <a href="index.html">Home</a>
+                    <a href="#">About</a>
+                    <a href="#">Clients</a>
+                    <a href="#">Contact Us</a>
+                </div>
+            </div>   
+            <div class="dropdown">
+                <h1 style="margin-left: 550px; color: black;">Signup Page</h1>                   
+            </div>
+            <div class="dropdown">
+                <button class="dropbtn" style="margin-left: 450px ;">Account</button>
+                <div class="dropdown-ctn" style="margin-left: 450px ;">
+                    <a href="signup.html">Sign up </a>
+                    <a href="#">Login</a>
+                    <a href="#">Track Orders</a>
+                    <a href="#">Wish List</a>
+                </div>
+            </div>                      
+        </div>        
+    </header>        
+    <body>    
+        <div class="signupbox">
+            <form> 
+                <h3 class="create">
+                    Create an Account
+                </h3>
+                Fullname:<br> 
```

Comment:
Well done Olumide!
Next time, try to style your input captions so as to make the UI more attractive. You can use the <label> to do this. I doubt if this form will look very good when viewed on the browser.
Job well done!
Kudos!!

Reply:
i will do that. Thank you